pkg/kanx/kanx_test.go

```diff
@@ -35,7 +35,7 @@ func tmpDir(c *C) string {
 	// limit the size of the address while retaining some of the properties
 	// of the original directory name
 	hs := adler32.Checksum([]byte(c.TestName()))
-	d, err := os.MkdirTemp("", fmt.Sprintf("%08x%.20s", hs, c.TestName()))
```

Comment:
Change to make these temp files easier to find.  Having the filename hash first can be really confusing when using `ls` and an alphabetical sort.